src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,51 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_USER_LOGIN_LENGTH = 6;
+    private static final int MAXIMUM_USER_LOGIN_LENGTH = 10;
+    private static final int MINIMUM_USER_PASSWORD_LENGTH = 6;
+    private static final int MAXIMUM_USER_PASSWORD_LENGTH = 15;
+    private static final int MINIMUM_USER_AGE = 18;
+    private static final int MAXIMUM_USER_AGE = 100;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("Can`t add user with parameter null");
```

Comment:
It would be a good idea to pass here a user that did not pass. Also you can do it in other exceptions